.teamcity/_self/projects/WebApp.kt

```diff
@@ -484,6 +484,7 @@ fun playwrightPrBuildType( targetDevice: String, buildUuid: String ): BuildType
 					export PLAYWRIGHT_BROWSERS_PATH=0
 					export TEAMCITY_VERSION=2021
 					export HEADLESS=true
+					export SAVE_AUTH_COOKIES=true
```

Comment:
Do we want to expose this as a TeamCity env variable so we can quickly turn it off? I think that can help diagnose future problems.

Reply:
Sounds like a good idea!